examples/local/README.md

```diff
@@ -0,0 +1,204 @@
+minikube. Also how to create a managed Kubernetes cluster with single master
+and single worker using the operator.
+
+All commands are assumed to be run from `examples/local` directory.
+
+
+## Preparing Templates
+
+All yaml files in this directory are templates. Before proceeding this guide
+all placeholders must be replaced with sensible values.
+
+- *CLUSTER_NAME* - Cluster's name.
+- *COMMON_DOMAIN* - Cluster's etcd and API common domain.
+- *COMMON_DOMAIN_INGRESS* - Ingress common domain.
+- *ID_RSA_PUB* - SSH public key to be installed on nodes.
+- *AWS_ACCESS_KEY_ID* - AWS access key.
+- *AWS_SECRET_ACCESS_KEY* - AWS secret.
+- *AWS_REGION* - AWS region.
+- *AWS_AZ* - AWS availability zone.
+- *AWS_AMI* - AWS image to be used on both master and worker machines.
+- *AWS_INSTANCE_TYPE_MASTER* - Master machines instance type.
+- *AWS_INSTANCE_TYPE_WORKER* - Worker machines instance type.
+
+Below is handy snippet than can be used to make that painless. It works in bash and zsh.
+
+```bash
+export CLUSTER_NAME="example-cluster"
+export COMMON_DOMAIN="internal.company.com"
+export COMMON_DOMAIN_INGRESS="company.com"
+export ID_RSA_PUB="$(cat ~/.ssh/id_rsa.pub)"
+export AWS_ACCESS_KEY_ID="AKIAIXXXXXXXXXXXXXXX"
+export AWS_SECRET_ACCESS_KEY="XXXXXXXXXXXXXXXXX/XXXXXXXXXXXXXXXXXXXXXX"
+export AWS_REGION="eu-central-1"
+export AWS_AZ="eu-central-1a"
+export AWS_AMI="ami-d60ad6b9"
+export AWS_INSTANCE_TYPE_MASTER="t2.medium"
+export AWS_INSTANCE_TYPE_WORKER="t2.medium"
+
+for f in *.tmpl.yaml; do
+    sed \
+        -e 's\${CLUSTER_NAME}\'"${CLUSTER_NAME}"'\g' \
+        -e 's\${COMMON_DOMAIN}\'"${COMMON_DOMAIN}"'\g' \
+        -e 's\${COMMON_DOMAIN_INGRESS}\'"${COMMON_DOMAIN_INGRESS}"'\g' \
+        -e 's\${ID_RSA_PUB}\'"${ID_RSA_PUB}"'\g' \
+        -e 's\${AWS_ACCESS_KEY_ID}\'"${AWS_ACCESS_KEY_ID}"'\g' \
+        -e 's\${AWS_SECRET_ACCESS_KEY}\'"${AWS_SECRET_ACCESS_KEY}"'\g' \
+        -e 's\${AWS_REGION}\'"${AWS_REGION}"'\g' \
+        -e 's\${AWS_AZ}\'"${AWS_AZ}"'\g' \
+        -e 's\${AWS_AMI}\'"${AWS_AMI}"'\g' \
+        -e 's\${AWS_INSTANCE_TYPE_MASTER}\'"${AWS_INSTANCE_TYPE_MASTER}"'\g' \
+        -e 's\${AWS_INSTANCE_TYPE_WORKER}\'"${AWS_INSTANCE_TYPE_WORKER}"'\g' \
+        ./$f > ./${f%.tmpl.yaml}.yaml
+done
+```
+
+- Note: `\` characters are used in `sed` substitubion to avoid escaping.
+
+
+## Cluster Certificates
+
+The easiest way to create certificates is to use local [cert-operator] setup.
+See [this guide][cert-operator-local-setup] for details.
+
+- Note: `CLUSTER_NAME` and `COMMON_DOMAIN` values must match ones used during
+  this guide.
+
+## Cluster-Local Docker Image
+
+The operator needs a connection to the K8s API. The simplest approach is to run
+as a deployment and use the "in cluster" configuration.
+
+In that case the Docker image needs to be accessible from the K8s cluster
+running the operator. For Minikube `eval $(minikube docker-env)` before `docker
+build`, see [reusing the Docker daemon] for details.
```

Comment:
This is the only one I don't get. Do you think it should be:

> For Minikube `eval $(minikube docker-env)` before `docker build`, [reusing the Docker daemon] for details.

I think it should be (missing "run"):

> For Minikube run `eval $(minikube docker-env)` before `docker build`, see [reusing the Docker daemon] for details.

PS. Thanks for your comments. ❤️ 👨‍🏫 

[reusing the docker daemon]: https://github.com/kubernetes/minikube/blob/master/docs/reusing_the_docker_daemon.md